addons/mail/static/src/core/common/thread_model.js

```diff
@@ -44,10 +44,16 @@ export class Thread extends Record {
     static new() {
         const thread = super.new(...arguments);
         Record.onChange(thread, ["state"], () => {
-            if (thread.state === "open" && !this.store.env.services.ui.isSmall) {
-                const cw = this.store.ChatWindow?.insert({ thread });
-                thread.store.chatHub.opened.delete(cw);
-                thread.store.chatHub.opened.unshift(cw);
+            if (thread.state === "open") {
+                if (!this.store.env.services.ui.isSmall) {
+                    const cw = this.store.ChatWindow?.insert({ thread });
+                    thread.store.chatHub.opened.delete(cw);
+                    thread.store.chatHub.opened.unshift(cw);
+                } else if (this.store.env.services["im_livechat.livechat"]) {
```

Comment:
Same remarks as the 18.1 PR, why limit it to livechat?
All pages in small screen could benefit from having it folded.

Reply:
In 18.1 the chat window sync is local so it's good.
In 18.0 however this is server-synced, and we definitely don't want to sync chat window state with mobile.
Even showing them as chat bubble is annoying, hence why this is best limited to limit the chat bubble in mobile for livechat visitor.